analyses/org.osate.analysis.flows/src/org/osate/analysis/flows/FlowLatencyAnalysisSwitch.java

```diff
@@ -835,41 +974,6 @@ public AnalysisResult invoke(ComponentInstance ci, SystemOperationMode som, bool
 		return results;
 	}
 
-	public AnalysisResult invoke(EndToEndFlowInstance etef) {
```

Comment:
Why did you remove this method? It could be used by a 3rd party plugin.

Reply:
I put it back